ozaria/site/components/common/ModalVictory.vue

```diff
@@ -0,0 +1,193 @@
+<script>
+  import BaseModal from './BaseModal'
+  import { mapActions, mapGetters } from 'vuex'
+  import { internationalizeConfig, getNextLevelForLevel, getNextLevelLink } from 'ozaria/site/common/ozariaUtils'
+  import utils from 'core/utils'
+
+  export default Vue.extend({
+    components: {
+      BaseModal
+    },
+    props: {
+      campaignHandle: {
+        type: String,
+        required: true
+      },
+      currentLevel: {
+        type: Object,
+        required: true
+      },
+      capstoneStage: {
+        type: String,
+        default: null
+      },
+      courseId: {
+        type: String,
+        default: null
+      },
+      courseInstanceId: {
+        type: String,
+        default: null
+      },
+      currentIntroContent: {
+        type: Object,
+        default: () => { return undefined }
+      },
+      introLevelComplete: {
+        type: Boolean,
+        default: undefined
+      },
+      goToNextDirectly: {
+        type: Boolean,
+        default: false
+      }
+    },
+    data: () => ({
+      nextLevelLink: ''
+    }),
+    computed: {
+      ...mapGetters({
+        levelsList: 'unitMap/getCurrentLevelsList'
+      }),
+      currentContent: function () {
+        return this.currentIntroContent || this.currentLevel.attributes || this.currentLevel
+      },
+      contentName: function () {
+        return this.currentContent.name
+      },
+      contentType: function () {
+        if (this.currentContent.ozariaType) {
+          return this.currentContent.ozariaType + ' level'
+        } else {
+          if (this.currentContent.contentType === 'cutscene-video') {
+            return 'cutscene'
+          }
+          if (this.currentContent.contentType === 'avatarSelectionScreen') {
+            return 'avatar selection'
+          }
+          return this.currentContent.contentType
+        }
+      },
+      learningGoals: function () {
+        const specificArticles = (this.currentContent.documentation || {}).specificArticles
+        const learningGoals = _.find(specificArticles, { name: 'Learning Goals' })
+        let learningGoalsText
+        if (learningGoals) {
+          learningGoalsText = internationalizeConfig(learningGoals).body
+        }
+        return learningGoalsText
+      }
+    },
+    async mounted () {
+      // TODO Use new audio system post-august launch
+      Backbone.Mediator.publish('audio-player:play-sound', { trigger: 'victory' })
```

Comment:
For my own understanding, if we change the trigger can we change the sound that plays? I think there will be different ozaria sounds that we'll need to incorporate.

Reply:
I am not sure I understand the question.

Reply:
Do we know the cost of changing the sound played for Ozaria?

Reply:
No as of now, we will have to consider that while implementing a new generic audio system